front_end/panels/rn_welcome/RNWelcome.ts

```diff
@@ -68,14 +67,23 @@ export class RNWelcomeImpl extends UI.Widget.VBox {
           ${i18nString(UIStrings.welcomeMessage)}
         </div>
         <div class="rn-welcome-links">
-          <x-link class="devtools-link" href="https://reactnative.dev/docs/debugging">
+          <x-link class="devtools-link" href=${this.getDocLink()}>
```

Comment:
- Big fan of this for the experiment `docLink` showing within Settings 💪🏻.
- Would prefer to leave the RN website decoupled and hardcoded here — motivation being we may update this to something else later.

Reply:
Very glad CDT has these built-in options! 🙌

To clarify, do you mean omitting reading the dynamic link from the RN Welcome page?

For OSS, landing this PR is no-op, i.e. the link will still be `https://reactnative.dev/docs/debugging` (and we can still change it here).

For internal use, we can also update this in the embedder script. Current internal diff points this to custom build tools instructions.

Happy to adjust the code and make this default constant more explicit if preferred

Reply:
Discussed offline. Reverting changes to RNWelcome. Only adding the link to settings